src/TextField/TextField.js

```diff
@@ -59,6 +59,7 @@ const getStyles = (props, context, state) => {
       outline: 'none',
       backgroundColor: 'rgba(0,0,0,0)',
       color: props.disabled ? disabledTextColor : textColor,
+      cursor: props.disabled ? 'not-allowed' : 'initial',
```

Comment:
'initial' is not supported by IE. This means that if I start with a disabled TextField, and later enable it, the cursor is still `not-allowed`. `auto` is also pretty aggressive for a library. Can the cursor style be removed entirely alternatively? http://www.w3schools.com/cssref/css_initial.asp